src/assets/translations/en-US.json

```
@@ -821,6 +821,12 @@
       "move to previous working day": "move to previous working day",
       "move to next meeting day": "move to next meeting day"
     },
+    "conditions": {
```

Biome (2.1.2)

[error] The key conditions was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)